### ISSUE TITLE
Incorrect example use of aria-label

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/auro-button#296]
### Description

See the following code:

https://github.com/AlaskaAirlines/auro-button/blob/f08e45934106d9a7edbc36a330dfdf879c336e20/apiExamples/iconOnly.html#L1-L9

In this and other examples, the author uses `aria-label` on the `auro-button` element. This is an incorrect use of the element's API.

The API is designed so that the attribute of `arialabel` is used on the root node, and then its value is reflected inside the shadow DOM for the `aria-label` attribute. 

We cannot use `aria-label` on the root node because the component doesn’t have an explicit `role` attribute, and the `aria-label` attribute is only valid on elements with a role that supports it. E.g. it would cause issues if we added `role="button"` on the `auro-button` element itself since it contains a semantically correct `button` inside the shadow DOM.  

The fix is to update the examples to follow the established API. 

### Additional context

<img width="1342" alt="Image" src="https://github.com/user-attachments/assets/aafbe212-9939-467c-b8d0-93a931774c82" />
